chia/types/blockchain_format/serialized_program.py

```diff
@@ -1,5 +1,56 @@
 from __future__ import annotations
 
+from typing import Any, TypeVar
+
 import chia_rs
 
+from chia.types.blockchain_format.program import Program
+
 SerializedProgram = chia_rs.Program
+
+
+def to_program(self) -> Program:
+    """
+    Convert the SerializedProgram to a Program object.
+    """
+    return Program.from_bytes(bytes(self))
+
+
+SerializedProgram.to_program = to_program
+
+
+@classmethod
+def from_program(cls, program: Program):  # noqa
```

Comment:
what does it mean for a free function to be a `@classmethod`?

Reply:
That it will take a class as described in the `cls` method but what that is is ambiguous at the moment

The same way `@classmethod` is flexible with children